CMakeLists.txt

```diff
@@ -68,6 +70,18 @@ endif()
 
 add_feature_info(ZeroHourStuff GENZH_BUILD_ZEROHOUR "Build Zero Hour code")
 add_feature_info(GeneralsStuff GENZH_BUILD_GENERALS "Build Generals code")
+add_feature_info(InternalBuild GENZH_BUILD_INTERNAL "Building as an \"Internal\" build")
+add_feature_info(DebugBuild GENZH_BUILD_DEBUG "Building as a \"Debug\" build")
```

Comment:
Is the debug feature automatically selected when picking Debug configuration in Visual Studio solution?